src/components/vuestic-components/va-input/VaInput.demo.vue

```diff
@@ -6,6 +6,13 @@
         placeholder="Name"
       />
     </VbCard>
+    <VbCard title="autofocus">
```

Comment:
That's native input attribute and there is no reason to test it.
We don't bring any new behavior in.